### PR TITLE
docs: Add "Hello World" example

### PR DESCRIPTION
**Pull request by jlebon**
Let's get practical faster in the manual and have a simple "Hello World"
example right off the bat to hopefully make it easier to grok how OSTree
works.

Also some minor tweaks on wording around comparisons to git.

### CODE REVIEW

**Comment by miabbott** [comment]
I really like including this kind of example in the docs, since it shows how `ostree` can be used for things other than just operating systems.  Do you think it could be expanded (maybe in another section/PR) to highlight the inherent de-duplication of the hardlink model?

**Comment by takluyver** [comment]
Thanks, this already helps me understand.

Is it useful to show how multiple repositories interact? Looking at the `ostree` command on my system, I see that `ostree pull` exists, but `ostree push` does not.

File: docs/manual/introduction.md [hunk 1]
```diff
@@ -11,15 +11,78 @@ replicating them to clients.
...
+## Hello World example
+
+OSTree is mostly used as a library, but a quick tour of using its
+CLI tools can give a general idea of how it works at its most
+basic level.
+
+You can create a new OSTree repository using `init`:
+
+```
+$ ostree --repo=repo init
```
Comment (cgwalters) [inline docs/manual/introduction.md]: I'm OK with this, but maybe note that ostree has different repository modes and to read the manual for more information?

**Comment by cgwalters** [comment]
@rh-atomic-bot r+ 23151cf

**Comment by jlebon** [comment]
@rh-atomic-bot retry

**Comment by jlebon** [comment]
@rh-atomic-bot retry

**Pull request closed without merging**